pkg/ring/ring.go

```diff
@@ -25,16 +25,18 @@ const (
 
 	// ConsulKey is the key under which we store the ring in consul.
 	ConsulKey = "ring"
+
+	maxExpectedReplicationSet = 5 // used to pre-allocate some arrays
```

Comment:
Can we put this in the file where it is used?

Also, IIUC, this could be called "max replication factor plus one" or something similar?

Finally, sentences for comments pls.

Reply:
The code returns more ingesters than the replication factor under circumstances I don't fully understand. I'm not at all certain that "plus one" is correct otherwise I would have made the constant 4.